pyclang/idf_extension.py

```diff
@@ -24,6 +32,15 @@ def check_clang_toolchain(subcommand_name, args):
                f'set to "{toolchain}".'), file=sys.stderr)
 
     def call_runner(subcommand_name, ctx, args, **kwargs):
+        if not check_esp_clang():
+            print(
+                'ERROR: Espressif Clang was not found on the system.\n'
+                'For installation instructions, visit the documentation:\n'
+                'https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-clang-tidy.html',
```

Comment:
Since we are now in the IDF extention part, we can suggests to run `idf.py docs tools/idf-clang-tidy.html` which will resolve the documentation to the current ESP-IDF version.